torch/testing/_internal/distributed/rpc/cuda_distributed_rpc_test.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
Add comment with link to example on the site

Reply:
okay added comment with example link.